Formula/saml2aws.rb

```diff
@@ -0,0 +1,24 @@
+class Saml2aws < Formula
+  desc "Login and retrieve AWS temporary credentials using a SAML IDP"
+  homepage "https://github.com/Versent/saml2aws"
+  url "https://github.com/Versent/saml2aws/archive/v1.8.4.tar.gz"
+  sha256 "3561a58006a27e4e166e3798a2c38b5b4c746426e9a760e4fdb44cfe654a5098"
+
+  depends_on "go" => :build
+  depends_on "glide" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GLIDE_HOME"] = HOMEBREW_CACHE/"glide_home/#{name}"
+    dir = buildpath/"src/github.com/versent/saml2aws"
+    dir.install Dir["*"]
+    cd dir do
+      system "glide", "install"
+      system "go", "build", "-o", bin/"saml2aws", "./cmd/saml2aws"
+    end
+  end
+
+  test do
+    system bin/"saml2aws", "--version"
```

Comment:
This needs to be a test that exercises and verifies the functionality of the software

Reply:
There's no apparent good way of doing that, as it's a fairly limited tool. Two of the three supported commands require server communication (AWS/idP), and the third command could overwrite a config file on the user's machine. Is the latter acceptable, or do you want me to add a comment à la [docker-credential-helper.rb#L32](https://github.com/Homebrew/homebrew-core/blob/master/Formula/docker-credential-helper.rb#L32)?

Reply:
>and the third command could overwrite a config file on the user's machine

the tests run in a sandbox so that's not really possible